executor/src/main/java/com/redhat/service/bridge/executor/ExecutorConfigProvider.java

```diff
@@ -0,0 +1,17 @@
+package com.redhat.service.bridge.executor;
+
+import java.util.Set;
+
+public interface ExecutorConfigProvider {
```

Comment:
I think the name is misleading because it mentions `Config` while it is a provider of the actual `Executor` instances so I would use `ExecutorProvider`. Wdyt?